config/ingest.go

```diff
@@ -16,14 +20,19 @@ type Ingest struct {
 	// or a chain of advertisement entries.  The value is an integer string
 	// ending in "s", "m", "h" for seconds. minutes, hours.
 	SyncTimeout Duration
+
+	// The recursion depth limit when syncing advertisement entries.
+	// The value 0 means no limit.
+	EntriesRecursionLimitDepth int64
 }
 
 // NewIngest returns Ingest with values set to their defaults.
 func NewIngest() Ingest {
 	return Ingest{
-		PubSubTopic:    "/indexer/ingest/mainnet",
-		StoreBatchSize: 256,
-		SyncTimeout:    Duration(2 * time.Hour),
+		PubSubTopic:                "/indexer/ingest/mainnet",
+		StoreBatchSize:             256,
+		SyncTimeout:                Duration(2 * time.Hour),
+		EntriesRecursionLimitDepth: 400,
```

Comment:
Should this be called `EntriesRecursionLimit` or `EntriesDepthLimit`?  Seems like Recursion and Depth or redundant terms.